common/perception_utils/include/perception_utils/transform.hpp

```diff
@@ -15,24 +15,26 @@
 #ifndef PERCEPTION_UTILS__TRANSFORM_HPP_
 #define PERCEPTION_UTILS__TRANSFORM_HPP_
 
-#include "geometry_msgs/msg/transform.hpp"
+#include <geometry_msgs/msg/transform.hpp>
+
+#include <boost/optional.hpp>
 
 #include <tf2_ros/buffer.h>
 #include <tf2_ros/transform_listener.h>
-
 #ifdef ROS_DISTRO_GALACTIC
 #include <tf2_geometry_msgs/tf2_geometry_msgs.h>
 #else
 #include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
 #endif
 
-#include "boost/optional.hpp"
-
 #include <string>
 
-namespace
+// To avoid unnamed namespace function in include file and to write template function in include
+// file, avoid unnamed namespace
+// "detail" name is used in boost library as same reason.
```

Comment:
I personally think this comment can be dropped because `detail` is a common style. But I'll leave it to you.

Reply:
Fixed at [61f3767](https://github.com/autowarefoundation/autoware.universe/pull/2028/commits/61f3767523b4060d1cafbd34657685be9f8e2f9a)!